### PR TITLE
Fix single lesson/review message; Account page redirect

### PR DESCRIPTION
**Pull request by imasaru**
Do with this what you may... I am learning Kotlin, and I needed an experiment, so I chose this extension because JavaScript and Kotlin are similar (as far as I know). It took me a few hours for 16 lines (programming newbie life), but I did it! If there's a better way to implement this, I would love to see it!

I wanted to start with something small... I may add bigger things in the future 😃

### CODE REVIEW

**Comment by imasaru** [comment]
@naei If you can, please fix my 2nd commit for me... I can't figure out how to make the redirect work completely. It still opens the Account page when I'm already on it.

File: js/helper.js [hunk 1]
```diff
@@ -105,13 +105,25 @@ function requestUserData(notify, callback) {
       if (notify === true && currentData.refreshInterval != 0){
           var notified = false;
           if (nbReviews > 0 && nbReviews != currentData.nbReviews) {
-            createNotification("You have " + nbReviews +" reviews available.", "https://www.wanikani.com/review", "reviews");
-            notified = true;
+            if (currentData.nbReviews === 1) {
```
Comment (naei) [inline js/helper.js:108]: `nbReviews` is the value you want here.

File: js/helper.js [hunk 2]
```diff
@@ -105,13 +105,25 @@ function requestUserData(notify, callback) {
...
+            // special case of a single review
+              createNotification("You have " + nbReviews +" review available.", "https://www.wanikani.com/review", "reviews");
```
Comment (naei) [inline js/helper.js:110]: No need to duplicate everything. :) You can only change the word with something like `var reviewTxt = (nbReviews > 1) ? "reviews" : "review";` then use it in the message.

File: js/home.js [hunk 3]
```diff
@@ -6,6 +6,13 @@ window.onload = function() {
   // display info message if the user is coming for the first time
   if (wkUserData.userPublicKey === undefined || wkUserData.userPublicKey == ""){
     document.querySelector(".info").style.display = 'inline';
+    if (window.location.href.includes('wanikani.com/settings/account')) {
+      alert("Please enter your API Key.")
+    } else {
+      alert("Redirecting to WaniKani...")
+      window.open('https://www.wanikani.com/settings/account/#public-api-key',
```
Comment (naei) [inline js/home.js:13]: It's a good intention, but too much intrusive :) People don't want to be redirected to a new page without their consent. However you could add a link on the ["API Key" text](https://github.com/naei/wanikani-companion/blob/master/html/settings.html#L29), which will redirect the user to the relevant URL on clicking on it.

**Review by naei — COMMENTED** [review]
Thank you for your contribution @imasaru. I put my suggestions in the review. Have fun coding ;)

File: js/home.js (hunk 3 above)
Comment (imasaru) [reply js/home.js:13]: I thought about intrusiveness.. In fact, my idea at first was to have it automatically fetch the API key! :rofl: I thought that was too intrusive. Now that I think about an automatic redirect, that seems intrusive also. 

I just got a free lesson on UX design!

File: js/helper.js (hunk 2 above)
Comment (imasaru) [reply js/helper.js:110]: I'm glad there's a better way. Duplicating everything seemed like a bad solution...

Thanks!

File: js/helper.js (hunk 1 above)
Comment (imasaru) [reply js/helper.js:108]: You refer to the green line 108, right? I see now - nbReviews makes more sense.

File: js/helper.js (hunk 2 above)
Comment (imasaru) [reply js/helper.js:110]: I don't quite understand the function of the colon, though. It's essentially saying, "replace 'review' with 'reviews'", right?

Comment (naei) [reply js/helper.js:110]: [Not exactly](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) ;)

Comment (imasaru) [reply js/helper.js:110]: Wow... I need to bookmark the MDN website. They seem to have a lot of resources. Thanks!

**Pull request closed without merging**